index.html

```diff
@@ -12,20 +12,26 @@
 <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@300;400;500;700;800&family=Poppins:wght@400;700&family=Roboto:wght@300;400;700&display=swap" rel="stylesheet">
 </head>
 <body>
+  <div id="menu-section" class="menu-section">
+          <button id="menu-toggle" class="menu-toggle"><i class="fa-solid fa-xmark"></i></button>
+        <ul class="menu-list">
+          <li class="menu-item"><a href="#portfolio">Portfolio</a></li>
+          <li class="menu-item"><a href="#about">About</a></li>
+          <li class="menu-item"><a href="#contact">Contact</a></li>
+        </ul>
```

Comment:
- [ ] as part of the requirements the mobile menu should only be visible in the mobile view , kindly fix the issue where it stays visible for large screens 🙏 

![fxxxxxxx](https://user-images.githubusercontent.com/89282221/217260607-9359a8dc-8af9-4f37-b768-139115fcc7c2.gif)